doc/scenario.md

```diff
@@ -86,17 +89,23 @@ In addition to the default scenario (see [steps](#steps)), there are also the fo
   - [configure_instance](#configure_instance)
   - [restart_instance](#restart_instance)
   - [wait_instance_started](#wait_instance_started)
+  - [wait_all_instances_alive](#wait_all_instances_alive)
+  - [wait_cluster_has_no_issues](#wait_cluster_has_no_issues)
   - [cleanup](#cleanup)
 - `configure_topology`:
   - [connect_to_membership](#connect_to_membership)
   - [edit_topology](#edit_topology)
   - [cleanup_expelled](#cleanup_expelled)
+  - [wait_all_instances_alive](#wait_all_instances_alive)
+  - [wait_cluster_has_no_issues](#wait_cluster_has_no_issues)
   - [cleanup](#cleanup)
 - `configure_app`:
   - [configure_auth](#configure_auth)
   - [configure_app_config](#configure_app_config)
   - [bootstrap_vshard](#bootstrap_vshard)
```

Comment:
You forget about variables for this task

Reply:
The same as above - in separated PR.